src/mlpack/methods/lars/lars.cpp

```diff
@@ -538,3 +538,22 @@ void LARS::CholeskyDelete(const size_t colToKill)
     matUtriCholFactor.shed_row(n);
   }
 }
+
+double LARS::ComputeError(const arma::mat& matX,
+                          const arma::rowvec& y,
+                          const bool rowMajor)
+{
+  arma::rowvec u;
+
+  if (rowMajor)
+  {
+    u = trans(matX * betaPath.back());
+    return arma::accu(arma::pow(y - u, 2.0));
```

Comment:
I guess we could even remove the extra copy here, if we use: `return arma::accu(arma::pow(y - trans(matX * betaPath.back()), 2.0));`, sorry that wasn't clear from the first comment.

Reply:
It's ok @zoq and thanks for the review